src/products/accounts/Account.java

```diff
@@ -48,11 +58,11 @@ public void setAccountBalance(float accountBalance) {
 		this.accountBalance = accountBalance;
 	}
 
-	public String getAccountCurrency() {
+	public Currency getAccountCurrency() {
```

Comment:
```suggestion
	public String getAccountCurrency() {
	    return accountCurrency.getName();
	}
```
You can use it like this.

Reply:
ok